### PR TITLE
feat: Time format switch

### LINKED ISSUE #697: Time format switch
### Please describe the problem that your request should resolve.
Provide better way to display time

### Describe the solution you'd like

In tables:

https://github.com/aeternity/aescan/assets/15363559/046a540e-6280-4070-ba56-914dd4d6cad9

In detail
image

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #697

## Demo

https://github.com/aeternity/aescan/assets/15363559/c3813de0-a70d-4e1d-874d-b42cd7f77198

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add time switcher to tables
- [x] unite time format in details
- [x] add clock icon

### CODE REVIEW

**Comment by janmichek** [comment]
Extra addition I made is adding clock icon as I saw this in referenced solutions.
Let me know what you think. Can be removed easily
image

**Comment by janmichek** [comment]
For unification in details, some refactoring needed to be done. Now it's separated into 2 rows. Height and time
image

Exception has been made in Oracle detail as there are 4 time entries. So there keeps on one line
image

Let me know if I should also separate it into block height and time per row

**Review by Liubov-crypto — COMMENTED** [review]
In general LGTM. 

I see that in almost all cases where time changes age, the alignment is slightly off (1 or 2 pixels). @janmichek could you check it please? 

t
al1
al

https://github.com/aeternity/aescan/assets/69896204/9c2271b1-4410-4b51-9830-8e4e70a72c7e

**Comment by Liubov-crypto** [comment]
When I opened it on the first time it has strange view and was placed inside grey bocks: 
https://github.com/aeternity/aescan/assets/69896204/626b5798-7df5-4e5a-a1e5-a1aad49631b3

after it's disappeared but aligning is not perfect yet:

**Comment by janmichek** [comment]
> When I opened it on the first time it has strange view and was placed inside grey bocks: https://github.com/aeternity/aescan/assets/69896204/626b5798-7df5-4e5a-a1e5-a1aad49631b3
> 
> after it's disappeared but aligning is not perfect yet:
> 
> 

I am still working on it and running few experiments. Please wait for another round of review. I will request it from you

**Comment by janmichek** [comment]
> In general LGTM.
> 
> I see that in almost all cases where time changes age, the alignment is slightly off (1 or 2 pixels). @janmichek could you check it please?
> 
> t al1 al
> 2024-04-09.1.44.46.mov

Fixed

File: nuxt.config.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { compileTemplate } from 'vue/compiler-sfc'
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
   srcDir: './src',
-  css: ['@/styles/main.css'],
+  css: ['@/assets/styles/main.css'],
```
Comment (janmichek) [inline nuxt.config.ts:7]: I moved styles to the initial Nuxt folder structure. Previous solution was a bit experimental

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Review by michele-franchi — COMMENTED** [review]
Good job, looks good but I would still keep the tooltips so when one value is selected you can also hover and see the other one.
I would add it back **except** where you already printed both absolute and relative time, for example:
image

**Comment by janmichek** [comment]
>  would still keep the tooltips so when one value is selected you can also hover and see the other one

Ok, no probs. Will do it.

File: src/utils/hints/contractsHints.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 export const contractsHints = {
   contract: 'A smart contract is a program that runs on the æternity blockchain. It\'s a collection of code (its functions) and data (its state) that resides at a specific address (contract id) on the æternity blockchain. Each smart contract has an account associated to it. Functions can be executed by any account via ContractCallTx and may change the smart contract\'s state. Smart contracts on the æternity blockchain are written in Sophia, a typed functional programming language.',
   contractId: 'Unique identifier of a smart contract instance that lives on the blockchain. The green icon indicates whether the smart contract has been verified.',
-  contractCreated: 'Keyblock height and exact date and time when the smart contract was created.',
+  contractCreatedHeight: 'Keyblock height  when the smart contract was created.',
```
Comment (michele-franchi) [inline src/utils/hints/contractsHints.js:4]:
```suggestion
  contractCreatedHeight: 'Keyblock height when the smart contract was created.',
```

File: src/utils/hints/namesHints.js [hunk 3]
```diff
@@ -10,14 +12,22 @@ export const namesHints = {
...
   ends: 'Keyblock height and estimated date and time when the auction ends.',
+  endsTime: 'Estimated date and time when the auction ends.',
+  endsHeight: 'Keyblock height when the auction ends.',
   expired: 'Keyblock height and estimated date and time when the name has been detached from the account. Name passed revoked period and can be claimed again.',
+  expiredTime: 'Estimated date and time when the name has been detached from the account. Name passed revoked period and can be claimed again.',
+  expiredHeight: 'Keyblock height  when the name has been detached from the account. Name passed revoked period and can be claimed again.',
   revoked: 'Keyblock height and estimated date and time when the name has been detached from the account. Name is in revoked state and can be claimed again after a fixed timeout of 2016 keyblocks after expiration.',
+  revokedTime: 'Estimated date and time when the name has been detached from the account.',
+  revokedHeight: 'Keyblock height  when the name has been detached from the account. Name is in revoked state and can be claimed again after a fixed timeout of 2016 keyblocks after expiration.',
```
Comment (michele-franchi) [inline src/utils/hints/namesHints.js:22]:
```suggestion
  revokedHeight: 'Keyblock height when the name has been detached from the account. Name is in revoked state and can be claimed again after a fixed timeout of 2016 keyblocks after expiration.',
```

**Review by michele-franchi — APPROVED** [review]
Found some extra spaces but looks good!

**Pull request merged**